app/assess/forms/comments_form.py

```diff
@@ -4,6 +4,5 @@
 
 
 class CommentsForm(FlaskForm):
-    
-        comment = TextAreaField("Comment",validators = [length(max=200)])
-        
\ No newline at end of file
+
+    comment = TextAreaField("Comment", validators=[length(max=200)])
```

Comment:
200 characters seems very small for a comment. Can we check with the designers, but I think they didn't want a limit possibly, so maybe set an arbitrarily large limit like 10000?

Reply:
This was a python reformatted file unrelated to the PR so unsure on why it's been set to 200, ill bump up the limit 👍